localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -429,22 +432,74 @@ void EKFLocalizer::updateSimple1DFilters(
   z_filter_.update(z, z_dev, pose.header.stamp);
   roll_filter_.update(rpy.x, roll_dev, pose.header.stamp);
   pitch_filter_.update(rpy.y, pitch_dev, pose.header.stamp);
+
+  if (twist_queue_.size() > 0) {
+    auto twist = twist_queue_.back();
+    double yaw_bias, obs_variance;
+    simpleEstimateYawBias(pose, *twist, yaw_bias, obs_variance);
+    if (obs_variance < 100) {
+      diagnostic_yaw_bias_filter_.update(yaw_bias, obs_variance, pose.header.stamp);
+    }
+    previous_ndt_pose_ = pose;
+    DEBUG_INFO(
+      get_logger(), "[1DEKF] yaw_bias = %f; [1DEKF] filtered_yaw_bias = %f", yaw_bias,
+      diagnostic_yaw_bias_filter_.get_x());
+  }
+}
+
+void EKFLocalizer::simpleEstimateYawBias(
```

Comment:
Would you consider making this estimation as isolated as possible from the current `EKFLocalizer` class, e.g. by
1. Making an individual node for this
2. Making an individual class e.g. `YawBiasEstimationModule` and hold it as a member variable in `EKFLocalizer`

Since current `EKFLocalizer` already have several functionalities which makes the node complicated, I want to maintain the base class as simple as possible.

Reply:
@kminoda 
Thanks for the suggestion.  In the latest commit (b02599e), I made Simple1DFilter and YawBiasEstimator individual classes in individual files, and Simple1DFilter as the base class of YawBiasEstimator.